cassandra-analytics-common/build.gradle

```diff
@@ -41,20 +41,21 @@ publishing {
 }
 
 dependencies {
-    implementation "org.slf4j:slf4j-api:${slf4jApiVersion}"
-    compileOnly "com.esotericsoftware:kryo-shaded:${kryoVersion}"
-    compileOnly "com.google.guava:guava:${guavaVersion}"
-    compileOnly "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
-    compileOnly "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
-    compileOnly "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
+    implementation("org.slf4j:slf4j-api:${slf4jApiVersion}")
+    compileOnly("com.esotericsoftware:kryo-shaded:${kryoVersion}")
+    compileOnly("com.google.guava:guava:${guavaVersion}")
+    compileOnly("com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}")
+    compileOnly("com.fasterxml.jackson.core:jackson-core:${jacksonVersion}")
+    compileOnly("com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}")
 
-    testImplementation "com.google.guava:guava:${guavaVersion}"
-    testImplementation "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
-    testImplementation "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
-    testImplementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
-    testImplementation("org.junit.jupiter:junit-jupiter-api:${project.junitVersion}")
-    testImplementation("org.junit.jupiter:junit-jupiter-params:${project.junitVersion}")
-    testImplementation("org.junit.jupiter:junit-jupiter-engine:${project.junitVersion}")
+    testImplementation("com.google.guava:guava:${guavaVersion}")
+    testImplementation("com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}")
+    testImplementation("com.fasterxml.jackson.core:jackson-core:${jacksonVersion}")
+    testImplementation("com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:${junitVersion}")
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
+    testImplementation("org.assertj:assertj-core:${assertjCoreVersion}")
```

Comment:
the actual change is this line, adding assertj-core dependency. Other lines in the file are changed to unify the dependency declaration style.